modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-registry-mirror_{context}"]
+= Configuring image registry repository mirroring
+Setting up container registry repository mirroring lets you:
```

Comment:
I'd split this information into a concept - the procedures are supposed to have very brief intro statements.

Reply:
I'm not sure if, to do this, I can just add the concept module into the existing assembly or if I need to create a new assembly that includes the concept and procedure. For now, I want to get this published, so I'll leave the text where it is (since moving it has no impact on how it is output) and deal with the separate concept module if I have time later in the week.

Reply:
I agree we don't want to impact the release here, but do think it should be updated post release. I can see the first part of this module being broken up into a single concept. Alternately, you could approach it as a Concept (About reg rep mirroring...) and a Reference and also perhaps a reference (Reg repo mirr attributes).